dohproxy/utils.py

```diff
@@ -338,7 +339,16 @@ def configure_logger(name='', level='DEBUG'):
     :param level: (optional) level of logging, default: DEBUG.
     :return: a logger instance.
     """
-    logging.basicConfig(format='%(asctime)s: %(levelname)8s: %(message)s')
+
+    if sys.stdout.isatty():
+        logging.basicConfig(
+            format='%(asctime)s: %(name)s/%(levelname)s: %(message)s'
+        )
+    else:
+        # No TTY (e.g. running in service manager), do not display time
+        logging.basicConfig(
+            format='%(name)s/%(levelname)s: %(message)s'
+        )
```

Comment:
nit: In order to ensure that logging format stays consistent whether or not it runs in a TTY, can `%(name)s/%(levelname)s: %(message)s` become a constant that gets appended to `'%(asctime)s: ` for the TTY case, and otherwise is directly used as format?

Reply:
Done. I've actually conditionally prepended the time to a single log_format, so it looks even cleaner and only shows basicConfig once.